mage_ai/ai/llm_pipeline_wizard.py

```diff
@@ -188,9 +189,13 @@
 class LLMPipelineWizard:
     def __init__(self):
         repo_config = get_repo_config()
-        openai_api_key = repo_config.openai_api_key or os.getenv('OPENAI_API_KEY')
-        openai.api_key = openai_api_key
-        self.llm = OpenAI(openai_api_key=openai_api_key, temperature=0)
+        ai_config = repo_config.ai_config
+        if ai_config.get("model_type") == AIMode.OPEN_AI:
+            self.client = OpenAIClient()
+        elif ai_config.get("model_type") == AIMode.HUGGING_FACE:
+            self.client = HuggingFaceClient()
+        else:
+            print("Error: AI Mode is not available.")
```

Comment:
Raise exception here. Otherwise, it'll continue run the code with "None" client

Reply:
Make sense